lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOAccessControlListDAOScaleTest.java

```diff
@@ -94,7 +96,7 @@ public void before() throws NotFoundException, DatastoreException,
 
 			// Add each type
 			Set<ACCESS_TYPE> types = new HashSet<ACCESS_TYPE>();
-			for (ACCESS_TYPE type : ACCESS_TYPE.values()) {
+			for (ACCESS_TYPE type : getAccessListForEntity()) {

```

Comment:
the test can use the same map that is used to validate what can be added to an entity.